doc/observable.md

```diff
@@ -308,11 +308,11 @@ A `subscribe` call is simply a way to start an "Observable execution" and delive
 
 The code inside `Observable.create(function subscribe(observer) {...})` represents an "Observable execution", a lazy computation that only happens for each Observer that subscribes. The execution produces multiple values over time, either synchronously or asynchronously.
 
-There are three types of values an Observable Execution can deliver:
+There are three types of events, or notifications, that an Observable can execute:
```

Comment:
We don't use this language "executing notifications" anywhere, and it brings different connotations, it may be hinting something that doesn't actually happen. Observables "deliver" values/notifications/events, they don't "execute" them. I'd recommend reverting.